protoc_plugin/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## 19.0.3-dev
 
 * Ignore `annotate_overrides` in generated files.
+* Requires sdk 2.3.0
```

Comment:
Are there more changes planned to go into some larger release for this package? It seems like the SDK jump to 2.3.0 is a pretty large change to put in the dev release, was wondering if it's worth a version bump

Reply:
I do not plan, but I have seen such changes in others, like https://github.com/dart-lang/protobuf/pull/394